saraho1123/currency-exchange
# Feature/initial setup

## Is this a fix or a feature?
Feature

## What is the change?
This is the initial setup for this app. 
+ installed needed packages
+ set up router in index 
+ set up component file structure
+ added initial code to render a very basic and unstyled app
+ included very brief pseudocode for what I plan for each component

## Where should the reviewer start?
+ See each component js file

## How should this be tested?
+ clone down repo
+ run `npm install` in terminal
+ `npm start` in the browser to initial app. There is not much to see yet!

## Relevant Tickets

###Screenshot (optional)
